README.md

```diff
@@ -36,8 +32,6 @@ To get a local copy up and running follow these simple example steps.
 
 ### Deployment
 
-
-
 ## Authors
 
 👤 **Author1**
```

Comment:
- [ ] **Required** Let's add here the correct `link` of the Author description & remove `Author2` as it is a solo project.

Reply:
I have changed the link to correct github url